.buildkite/pipeline-e2e-tests.yml

```diff
@@ -62,3 +63,17 @@ steps:
         command: |
           cd .buildkite/e2e/pipeline-gen && go build -o pipeline-gen
           cat ../nightly-main-matrix.yaml | ./pipeline-gen | buildkite-agent pipeline upload
+
+      # After nightly e2e tests, run a cleanup scripts to ensure dangling clusters are deleted
+      - label: ":broom: e2e test cluster cleanup"
```

Comment:
Would it be possible to have a dedicated pipeline? I was thinking of something that could be developed and tested as stand-alone, and triggered manually.